tests/data_out/data_out_base_vtk_03.cc

```diff
@@ -0,0 +1,53 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2006 - 2017 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+// Test high-order Lagrange VTK output for on a unit cube.
+// Added by Alexander Grayver
+
+#include <deal.II/grid/grid_generator.h>
+#include <deal.II/grid/tria.h>
+
+#include <deal.II/numerics/data_out.h>
+
+#include <string>
+
+#include "../tests.h"
+
+template <int dim>
+void
+check(std::ostream &log, unsigned cell_order)
+{
+  Triangulation<dim> triangulation;
+  GridGenerator::hyper_cube(triangulation);
+
+  DataOutBase::VtkFlags flags;
+  flags.write_higher_order_cells = true;
+
+  DataOut<dim> data_out;
+  data_out.set_flags(flags);
+  data_out.attach_triangulation(triangulation);
+  data_out.build_patches(cell_order);
+  data_out.write_vtk(log);
+}
+
+int
+main()
+{
+  std::ofstream logfile("output");
```

Comment:
Please replace this line with `initlog();`

You can then get the output file with `deallog.get_file_stream()`